app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -29,17 +29,38 @@
         <p><%= title %>: <span class="other-date definition"><%= date %></span></p>
       <% end %>
     <% end %>
-    <% if history.any? %>
+    <%
+      hidden_history =
+          local_assigns.include?(:hidden_history) ? hidden_history : []
+    %>
+    <% if history.any? || hidden_history.any? %>
```

Comment:
This would support having a page history header, but no (default) visible history. I think `history` should be required if providing `hidden_history` - or if we decide against that, then there should fixture covering that example.